### DISCUSSION TITLE
Anpassung der Darstellung

### DISCUSSION THREAD

**Discussion by hanskaas** [electricitymaps/electricitymaps-contrib#7560] [Ideas]
When displaying country-specific imports and exports, a meaningful scale should be chosen. Today it goes from 0 to 60 GW and has no graphical significance. Imports should be displayed between 0 in the middle and +/- 10 GW.

When you click on the bar display for imports or exports, the real value is displayed. But it is always only export or import. If you want to see the price at that time, you have to close and click again. But what is of interest is the price and the quantity at the same time, which is why the corresponding values should always be displayed with one click.

**Comment by VIKTORVAV99** [comment]
Could you translate this to English so everyone can participate in the discussion.

Also not sure if the online translation I used is accurate so I would be more comfortable creating an actual reply to an English text.
